android/app/src/main/cpp/app/organicmaps/util/StringUtils.cpp

```diff
@@ -82,4 +82,10 @@ Java_app_organicmaps_util_StringUtils_nativeGetLocalizedSpeedUnits(JNIEnv * env,
 {
   return jni::ToJavaString(env, platform::GetLocalizedSpeedUnits());
 }
+
+JNIEXPORT jdouble JNICALL
+Java_app_organicmaps_util_StringUtils_nativeLocalizedSpeedToMps(JNIEnv * env, jclass, jdouble speed)
+{
+    return measurement_utils::UnitsToMps(speed, measurement_utils::GetMeasurementUnits());
+}
```

Comment:
Should we have conversion from mph/kph to meters-per-hour here? Is `StringUtils` a good place for it?